lib/sass/calcite-web/components/_dropdown.scss

```diff
@@ -13,45 +13,66 @@
 }
 
   @mixin dropdown-link() {
+    @include box-sizing(border-box);
+    @include font-size(-2)
     position: relative;
+    padding: $baseline/2.5 $baseline/1;
     display: block;
-    @include box-sizing(border-box);
-    padding: $baseline/3;
-    @include font-size(-2);
     color: $darker-gray;
-    background-color: $white;
-    border-bottom: none;
+    font-weight: inherit;
     text-align: left;
-    border-left: none;
-    border-right: none;
-    border-top: 1px solid $lightest-gray;
-    white-space: nowrap;
     cursor: pointer;
-    width: 100%;
-    &:hover {
+    background-color: transparent;
+    overflow: visible;
+    &:before {
+      content: "•";
+      color: $gray;
+      position: absolute;
+      left: 10px;
+      opacity: 0;
+    }
+    &:hover, &:focus {
       background-color: $off-white;
-      text-decoration: none;
       color: $darker-gray;
+      text-decoration: none;
+      &:before {
+        opacity: 1;
+      }
     }
-    &.is-active, &:focus {
-      text-indent: -3px;
-      border-left: 3px solid $blue;
+    &.is-active {
+      font-weight: 600;
+      &:before {
+        opacity: 1;
+        color: $blue;
+      }
+      &:hover:before, &:focus:before {
+        color: $blue;
+      }
     }
     @if ($include-right-to-left) {
       html[dir="rtl"] & {
         text-align: right;
       }
-      html[dir="rtl"] &.is-active, html[dir="rtl"] &:focus {
-        border-left: none;
-        border-right: 3px solid $blue;
+      html[dir="rtl"] &.is-active:before, html[dir="rtl"] &:focus:before, html[dir="rtl"] &:hover:before {
```

Comment:
you don't need all these selectors. You're only declaring the `left` value on the `:before` pseudo element above, so that's the only one you need to correct in right to left. Just using 

```
html[dir="rtl"] &:before {
  left: unset;
  right: 10px;
}
```

should be sufficient.

Reply:
yup, done.